packages/builders/test/snapshots/orchestration-imports.test.js.md

```diff
@@ -167,6 +167,9 @@ Generated by [AVA](https://avajs.dev).
             bech32Prefix: Object @match:string {
               payload: [],
             },
+            cctpDestinationDomain: Object @match:kind {
+              payload: 'number',
+            },
```

Comment:
I don't understand why `cctpDestinationDomain` gets added to `CosmosChainInfoShape`; to reach cosmos chains, we don't need to use CCTP.